--- ./ecell/pyecell/ecell/LoggerStub.orig.py	2013-04-17 12:25:15.000000000 +0900
+++ ./ecell/pyecell/ecell/LoggerStub.py	2013-10-22 08:43:23.000000000 +0900
@@ -66,11 +66,17 @@
         else:
             self.theSimulator().createLogger( self.theFullPNString )
 
+    def remove( self ):
+        """
+        Deletes the logger created by create()
+        """
+        self.theSimulator().removeLogger( self.theFullPNString )
+
     def delete( self ):
         """
         Deletes the logger created by create()
         """
-        self.theSimulator().deleteLogger( self.theFullPNString )
+        self.remove()
 
     def exists( self ):
         """
