--- ./ecell/configure.ac.in.orig	2013-10-22 06:04:24.000000000 +0900
+++ ./ecell/configure.ac.in	2013-10-26 16:32:03.000000000 +0900
@@ -317,6 +317,7 @@
     pyecell/ecell/config.py
     frontend/Makefile
     frontend/session-monitor/Makefile
+    frontend/session-monitor/ecell3-session-monitor
     frontend/session-monitor/glade/Makefile
     frontend/session-monitor/plugins/Makefile
     frontend/model-editor/Makefile
