--- ./ecell/libecs/Polymorph.orig.hpp	2011-10-28 19:18:21.000000000 +0900
+++ ./ecell/libecs/Polymorph.hpp	2011-11-04 22:51:11.000000000 +0900
@@ -1151,21 +1151,21 @@
 #endif
 
 template<>
-inline typename boost::range_iterator< libecs::PolymorphValue::Tuple >::type
+inline boost::range_iterator< libecs::PolymorphValue::Tuple >::type
 begin< libecs::PolymorphValue::Tuple >( libecs::PolymorphValue::Tuple& r )
 {
     return static_cast< libecs::Polymorph* >( r );
 }
 
 template<>
-inline typename boost::range_const_iterator< libecs::PolymorphValue::Tuple >::type
+inline boost::range_const_iterator< libecs::PolymorphValue::Tuple >::type
 begin< libecs::PolymorphValue::Tuple >( libecs::PolymorphValue::Tuple const& r )
 {
     return static_cast< libecs::Polymorph const* >( r );
 }
 
 template<>
-inline typename boost::range_iterator< libecs::PolymorphValue::Tuple >::type
+inline boost::range_iterator< libecs::PolymorphValue::Tuple >::type
 end< libecs::PolymorphValue::Tuple >(
         libecs::PolymorphValue::Tuple& r )
 {
@@ -1173,7 +1173,7 @@
 }
 
 template<>
-inline typename boost::range_const_iterator< libecs::PolymorphValue::Tuple >::type
+inline boost::range_const_iterator< libecs::PolymorphValue::Tuple >::type
 end< libecs::PolymorphValue::Tuple >(
         libecs::PolymorphValue::Tuple const& r )
 {
