--- ./ecell/libecs/Polymorph.hpp.orig	2011-01-04 17:17:05.000000000 +0900
+++ ./ecell/libecs/Polymorph.hpp	2011-04-24 22:25:54.000000000 +0900
@@ -1145,6 +1145,8 @@
     typedef libecs::PolymorphValue::Tuple::value_type const* type;
 };
 
+namespace range_adl_barrier
+{
 template<>
 inline range_iterator< libecs::PolymorphValue::Tuple >::type
 begin< libecs::PolymorphValue::Tuple >( libecs::PolymorphValue::Tuple& r )
@@ -1175,7 +1177,7 @@
     return static_cast< libecs::Polymorph const* >( r ) + r.size();
 }
 
-} // namespace boost
+}} // namespace boost::range_adl_barrier
 
 namespace libecs
 {
