--- ./doc/samples/ga/ga.orig.py	2011-01-04 17:17:10.000000000 +0900
+++ ./doc/samples/ga/ga.py	2011-10-30 14:18:05.000000000 +0900
@@ -232,7 +232,7 @@
             self.__executeEliteStrategy()
 
             # clear all jobs
-            clearJob(jobid=-1)
+            clearAllSessionProxies()
 
 
             print "[%s]\t%s" %(self.theCurrentGeneration, \
@@ -317,7 +317,7 @@
                 anIndividual.setJobIDList( aJobIDList )
 
 
-        aJobNumber = len(getQueuedJobList())
+        aJobNumber = len( list( getQueuedSessionProxyList()))
 
         # -----------------------------------------------------
         # [2] Run ess files using SessionManager
@@ -335,7 +335,7 @@
             time.sleep(1)
 
             sys.stdout.write("\r")
-            sys.stdout.write("(%s/%s)" %(len(getFinishedJobList()),aJobNumber))
+            sys.stdout.write("(%s/%s)" %(len(list(getFinishedSessionProxyList())),aJobNumber))
 
             if aLoopCounter % 6 == 1:
                 sys.stdout.write(" .    ")
@@ -355,7 +355,7 @@
 
             if isFinished() == True:
                 sys.stdout.write("\r")
-                sys.stdout.write("(%s/%s)" %(len(getFinishedJobList()),aJobNumber))
+                sys.stdout.write("(%s/%s)" %(len(list(getFinishedSessionProxyList())),aJobNumber))
                 sys.stdout.write(" .....")
                 break
 
