--- ./ecell/frontend/session-monitor/plugins/PropertyWindow.orig.py	2012-01-21 21:40:02.000000000 +0900
+++ ./ecell/frontend/session-monitor/plugins/PropertyWindow.py	2012-01-23 13:15:03.000000000 +0900
@@ -349,6 +349,7 @@
                 self.setSelectedFullPN( aFullPN )
             else:
                 self['entryClassName'].set_text( "" )
+                self['entryClassName'].set_max_length(0)
                 self['entryFullID'].set_text( "" )
                 self['entryName'].set_text( "" )
 
@@ -637,11 +638,8 @@
                     return None
 
 
-        aFullPNString = createFullPNString(self.theSelectedFullPN)
-
         try:
             self.theSession.setEntityProperty( self.theSelectedFullPN, aValue ) 
-            lockCursor = self.lockCursor
             self.lockCursor = True
             self['theTreeView'].get_selection().select_iter( anIter )
             self.lockCursor = False
