--- ./ecell/libecs/scripting/ExpressionCompiler.orig.cpp	2011-10-28 19:18:14.000000000 +0900
+++ ./ecell/libecs/scripting/ExpressionCompiler.cpp	2011-11-23 22:10:00.000000000 +0900
@@ -41,11 +41,11 @@
 
 namespace libecs { namespace scripting {
 
-using namespace boost::spirit;
+using namespace BOOST_SPIRIT_CLASSIC_NS;
 using namespace libecs;
 using namespace libecs::math;
 
-typedef boost::spirit::tree_match<const char*> TreeMatch;
+typedef BOOST_SPIRIT_CLASSIC_NS::tree_match<const char*> TreeMatch;
 typedef TreeMatch::tree_iterator TreeIterator;
 
 //  VariableReferenceMethodProxy;
