--- ./ecell/pyecell/setup.py.in.orig	2010-01-16 11:23:46.000000000 +0900
+++ ./ecell/pyecell/setup.py.in	2011-07-23 14:54:52.000000000 +0900
@@ -10,6 +10,8 @@
 from numpy.distutils.command.build_py import build_py as _build_py
 from types import *
 
+sys.path.append('@PREFIX@/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages')
+
 include_dirs = [
     r'@DMTOOL_INCLUDE_DIR@',
     os.environ['INCLTDL'][2:],
