--- ./ecell/libecs/Util.orig.hpp	2011-10-27 08:21:55.000000000 +0900
+++ ./ecell/libecs/Util.hpp	2011-11-23 16:15:45.000000000 +0900
@@ -275,6 +275,14 @@
    If not, throw a RangeError exception.
 */
 
+/** @internal */
+LIBECS_API void throwSequenceSizeError( const size_t aSize, 
+                                        const size_t aMin, const size_t aMax );
+
+/** @internal */
+LIBECS_API void throwSequenceSizeError( const size_t aSize, const size_t aMin );
+
+
 template <class Sequence>
 void checkSequenceSize( const Sequence& aSequence, 
                         const typename boost::range_size< Sequence >::type aMin )
@@ -287,14 +295,6 @@
 }
 
 
-/** @internal */
-LIBECS_API void throwSequenceSizeError( const size_t aSize, 
-                                        const size_t aMin, const size_t aMax );
-
-/** @internal */
-LIBECS_API void throwSequenceSizeError( const size_t aSize, const size_t aMin );
-
-
 /**
    Form a 'for' loop over a STL sequence.
 
