--- ./ecell/frontend/session-monitor/ecell/ui/osogo/FileSelection.orig.py	2011-12-16 14:09:53.000000000 +0900
+++ ./ecell/frontend/session-monitor/ecell/ui/osogo/FileSelection.py	2012-01-23 10:22:56.000000000 +0900
@@ -1,4 +1,5 @@
 import gtk
+import os
 
 __all__ = [
     'FileSelection',
@@ -14,6 +15,7 @@
 
     def __init__(self, *arg, **kwarg):
         gtk.FileChooserDialog.__init__(self, *arg, **kwarg)
+        self.set_current_folder(os.getcwd())
         self.cancel_button = self.add_button(gtk.STOCK_CANCEL, gtk.RESPONSE_CANCEL)
         self.ok_button = self.add_button(gtk.STOCK_OK, gtk.RESPONSE_OK)
         self.ok_button.grab_default()
