--- ./ecell/frontend/session-monitor/ecell/ui/osogo/MainWindow.py.orig	2012-08-14 20:39:34.000000000 +0900
+++ ./ecell/frontend/session-monitor/ecell/ui/osogo/MainWindow.py	2012-08-14 20:37:46.000000000 +0900
@@ -174,7 +174,7 @@
         self.isStarted = False
         self.timerVisible = False
         self.theLastTime = 0
-        self.theLastRealTime = 0                
+        self.theLastRealTime = 0
         self.updateCount = 0
 
         # initialize Indicator
@@ -832,7 +832,7 @@
         # updates time
         aTime = self.theSession.getCurrentTime()
         self.theCurrentTime = aTime
-        self['time_entry'].set_text( str( self.theCurrentTime ) )
+        self['time_entry'].set_text( str( self.theCurrentTime ) if ( aTime == aTime ) else '' )  #Check NaN
         self['sec_step_entry'].set_text( str( self.theStepSizeOrSec ) )
 
         if ( self.SimulationButton.getCurrentState() == 'run' and
